CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [3.9.0](https://github.com/marklogic/mlxprs/releases/tag/v3.9.0)
+
+- New MarkLogic status view including high-level information about the currently connected MarkLogic server
+- Can now be used to connect to a MarkLogic server behind a proxy server or in the ML Cloud
```

Comment:
I think it's worth calling out the names of the config options, as it's simple to do so here without getting into detail - e.g. "Can use new `basePath` and `cloudApiKey` configuration options to connect to MarkLogic behind a proxy server or in MarkLogic Cloud" (always spell "MarkLogic" too so people don't think we mean "machine learning").